Cargo.toml

```diff
@@ -65,72 +68,117 @@ resolver = "2"
 # commit of TheDan64/inkwell, push a new tag which points to the latest commit,
 # change the tag value in this Cargo.toml to point to that tag, and `cargo update`.
 # This way, GitHub Actions works and nobody's builds get broken.
-inkwell = { git = "https://github.com/roc-lang/inkwell", branch = "inkwell-llvm-15", features = [ "llvm13-0" ] }
+inkwell = { git = "https://github.com/roc-lang/inkwell", branch = "inkwell-llvm-15", features = ["llvm13-0"] }
```

Comment:
Kinda scary how many dependencies the compiler has. To be fair, these are runtime, dev, and build dependencies, but it is still a ton. 

I think it is nice to have them all in one place with guaranteed version matching. Also, gives us one place to look when thinking of replacing heavy dependencies with lighter weight counter parts.